apps/core/scripts/generate.cjs

```diff
@@ -42,11 +44,18 @@ const getEndpoint = () => {
 };
 
 const generate = async () => {
-  await generateSchema(getEndpoint(), {
+  await generateSchema({
+    input: getEndpoint(),
     headers: { Authorization: `Bearer ${getToken()}` },
+    output: join(__dirname, '../schema.graphql'),
```

Comment:
This API will use the current working directory if this is `undefined`. When being used in CI, it'll output the files at the root which will cause the next command to error out.